packages/react-error-overlay/src/overlay.js

```diff
@@ -156,6 +160,10 @@ function crash(error: Error, unhandledRejection = false) {
   }
   consumeError(error, unhandledRejection, CONTEXT_SIZE)
     .then(ref => {
+      if (ref == null) {
+        console.warn('react-error-overlay ignored:', error);
```

Comment:
Oh. I think in this case it's fine not to print anything. IMO let's silently leave the browser behavior in this case and see what people say.